api/RTCIceCandidate.json

```diff
@@ -124,6 +124,253 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "candidateInfo_parameter": {
```

Comment:
Sorry for the confusion. The constructor takes a dictionary of options as a parameter. The question "what should the parameter to the constructor be called".

TLDR: Can I use  `candidateInfo` as the name for this parameter to match MDN docs, ignoring the different name used in the spec (`candidateInitDict`.) and for the same options object passed to `addIceCandidate()` (`candidate`).

----

The long version

Some of the options (sorry for repetition):
- [In MDN](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate/RTCIceCandidate#parameters) this parameter is called `candidateInfo`. That is what I have chosen for the name of the parameter in BCD.
- [The spec](https://w3c.github.io/webrtc-pc/#dom-rtcicecandidate-constructor) just refers to it by `candidateInitDict`. I don't want to call something Dict while we're busy removing the dict terminology. But some might think this useful as a clear mapping for people reading the spec.
- `candidate` is the name of the parameter passed to the [RTCPeerConnection.addIceCandidate()](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/addIceCandidate#parameters) used in both the spec and MDN. This is the same dictionary as passed to the constructor - though it is described differently. I don't like this name because it ends up being hard to describe whether you mean the parameter `candidate`, the value `candidate.candidate`, the string `candidate` etc. I mention it because it is the same options object, but I don't want to use it here. And I would prefer not to use in in MDN or `addIceCandidate()`. So kind of a "question" - more for when doing BCD for `addIceCandidate()`

Reply:
Ahh, OK. I think I get it now: this constructor and `addIceCandidate()` both accept the same dict as a parameter, but the spec (and the two pages) use a different name for the two parameters. Bummer. I hate all the options, which, as I see them, are:

1. Follow the spec in each case (i.e., use `candidateInitDict` and `candidate`). Truest to the spec, but harder to reuse the text in both places and mentions dicts (though I think that's not too harmful in and of itself).
2. Follow the spec in one case and mirror the other (i.e., `candidate` and `candidate` — doesn't make sense to use the `init` terminology since only one is `init`-ing). Easy on authors, true-ish to the spec, but confusing to readers in the `candidate.candidate` case.
3. Follow the spec nowhere by using with `candidateInfo` in both places. Easy on authors and readers, but departs from the spec. For both option 2 and 3, we might someday end up reverting this, if we ever start validating content against the IDL.

Reluctantly, I accept option 3. Maybe we could aslo open an issue against the spec to ask the editors to rename the parameter in both places as a courtesy to beleaguered technical writers everywhere?

Reply:
Oh, I guess the actual action in this PR: can you change the notes in the parent feature to match the `candidateInfo` name? That will at least eliminate one of the names.

(I guess you could rename `candidate` to `candidateInfo` for `addIceCandidate()` too, but only if you're going to do the content work as well. We don't need to drag everything into the scope of this PR, unless you're excited to do so.)